yarn.lock

```diff
@@ -0,0 +1,1105 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
These configs in the root are not necessary, only need to add them in the `react/yarn.lock`.